catalog/region-specific-policies.yaml

```diff
@@ -0,0 +1,23 @@
+- sid: "DenyRegionUsage"
+  effect: "Deny"
+  not_actions:
+    - "cloudfront:*"
+    - "iam:*"
+    - "route53:*"
+    - "support:*"
+    - "organizations:*"
+    - "waf:*"
+    - "budgets:*"
+    - "globalaccelerator:*"
+    - "cur:*"
+    - "ce:*"
+    - "directconnect:*"
+  condition:
+    - test: "StringNotEqualsIgnoreCase"
+      variable: "aws:RequestedRegion"
+      values:
+        # The regions that can't be used for deploying infrastructure
+        - "${region1_lockdown}"
```

Comment:
It's possible to iterate, so why do we fix it? 

https://discuss.hashicorp.com/t/template-over-map/5102

Reply:
Or just do: (it doesn't need to be one-per-line)
```
values: [ ${join(",", $regions)} ]
```

Reply:
but this is yaml using the config module, can it run tf code not the template yaml files?

Reply:
rule deleted